benchmark_runner/workloads/windows_vm.py

```diff
@@ -25,8 +25,8 @@ def run(self):
         :return:
         """
         try:
-            if self._run_type == 'test_ci':
-                self._es_index = 'windows-test-ci-results'
+            if self._run_type in ('test_ci', 'chaos_ci'):
+                self._es_index = f"windows-{self._run_type.replace('_', '-')}-results"
```

Comment:
There are a lot of places with this code, and this technical debt is piling up.  I'll approve this, but there are entirely too many places in the code where this code is present.